omni-relayer/src/config.rs

```diff
@@ -26,6 +27,7 @@ pub struct Near {
     pub rpc_url: String,
     pub token_locker_id: AccountId,
     pub credentials_path: Option<String>,
+    pub relayer_address_on_evm: OmniAddress,
```

Comment:
Why relayer_address_on_evm in Near Config not in EVM Config?

Reply:
Makes sense, I'll fix it. But can we please rename it to `relayer_address`. I don't see a point of using config.**evm**.XXX_on_**evm**

Reply:
Fixed in 068b62c17626684187b8efbe99e76265bec3e6b3